webdriver-spec.html

```diff
@@ -1231,6 +1237,15 @@
  </tr>
 
  <tr>
+  <td><dfn>no such cookie</dfn>
```

Comment:
In Gecko we typically exhaust the iterator in the [cookie manager](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XPCOM/Reference/Interface/nsICookieManager) to serialise a list of cookies. Since we for the suggested _Get Named Cookie_ match on a user-given path we will know if our sequence of cookies has a cookie with that path or not. Based on that we can choose to error.